core/transport/src/main/java/alluxio/grpc/PMode.java

```diff
@@ -675,5 +675,16 @@ public PMode parsePartialFrom(
     return DEFAULT_INSTANCE;
   }
 
+  static public Bits parseOwnerBits(long mode) {
```

Comment:
This file `PMode.java` is auto generated by the protocol buffer compiler which we should not touch

Reply:
Thanks for your comment. I have fixed it.